tests/integration/sourcemap_monkey_integ_test.py

```diff
@@ -0,0 +1,133 @@
+from configparser import ConfigParser
+from importlib import import_module
+from pathlib import Path
+from unittest import mock
+
+import pytest
+
+"""
+This file monkey-patches ConfigParser in order to enable source mapping
+and test the results of source mapping various PyTeal dapps.
+"""
```

Comment:
TODO: Shall I add a test that asserts that the fixtures here are compatible with the pre-existing fixtures (for greater confidence that there are no regressions)

Reply:
no?

Reply:
Will investigate.

Reply:
https://github.com/algorand/pyteal/pull/650/commits/13bfc3b7470800c34a0c9e7f190a15881ae412fb